packages/runtime-core/__tests__/helpers/createSlots.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { Slot } from '../../src/componentSlots'
+import { createSlots } from '../../src/helpers/createSlots'
+import { createVNode } from 'vue'
+
+describe('createSlot', () => {
+  it('should return a slot', () => {
+    const record: Record<string, Slot> = {}
+    const slot = () => createVNode(Object)
```

Comment:
```suggestion
    const slot = () => []
```

Let's return an empty array.

Reply:
Just an empty array? That's all it takes? Oh. Okay.

Reply:
An empty array is a valid `VNodeChildren` value, and the functionality of `createSlots` should not depend on the contents of a slot, so it's a good mock.

Reply:
Nice, thanks. I think I get it now, and I just pushed a passing test. I'll keep going. Can I ping you when I think it's ready?